api/database/db.ts

```diff
@@ -0,0 +1,18 @@
+
+import pg from 'pg';
+import 'dotenv/config';
+
+// Placeholder database - this has the basic data that I parsed playing around with bitcoin blockchain data
+// Talk with the team about creating a new database?
```

Comment:
farhad said he'd do the initial db setup, will be pretty easy to just plug it in on these routes and queries